core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7504NotWarnUnsupportedReportPluginsTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.maven.it;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.maven.it.util.ResourceExtractor;
+
+/**
+ * Test for
+ * <a href="https://issues.apache.org/jira/browse/MNG-7504">MNG-7504</a>
+ *
+ * Warning about unsupported reportPlugins should not be printed for m-site-p.
+ *
+ * @author Slawomir Jaranowski
+ */
+public class MavenITmng7504NotWarnUnsupportedReportPluginsTest extends AbstractMavenIntegrationTestCase
+{
+    private static final String PROJECT_PATH = "/mng-7504-warn-unsupported-report-plugins";
+
+    public MavenITmng7504NotWarnUnsupportedReportPluginsTest()
+    {
+        super( "[3.9.0]" );
```

Comment:
I think you can even drop the brackets.

Reply:
Stupid question, why only 3.9.0? My understanding:
* 3.9.0 is suppressing this
* 4.0.0 does not even invoke

In both cases the warning should be visible, no?

Reply:
currently only 3.9.0 has a fix - https://github.com/apache/maven/pull/759
4.0.0 - still print warning MNG-7505